portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -21,11 +32,44 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     System.out.println("Name: " + name);
     System.out.println("Contact Method: " + contactMethod);
     System.out.println("Contact Information: " + contactInformation);
+ 
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Contact");
```

Comment:
Consider defining constants for strings that are used across multiple datastore interactions.